homeassistant/components/tautulli/sensor.py

```diff
@@ -82,62 +269,55 @@ async def async_setup_entry(
 ) -> None:
     """Set up Tautulli sensor."""
     coordinator: TautulliDataUpdateCoordinator = hass.data[DOMAIN]
-    async_add_entities(
+    entities: list[TautulliSensor | TautulliSessionSensor] = [
         TautulliSensor(
             coordinator,
             description,
         )
         for description in SENSOR_TYPES
-    )
+    ]
+    if coordinator.users:
+        for user in coordinator.users:
+            if user.username != "Local":
+                for description in SESSION_SENSOR_TYPES:
+                    _description = deepcopy(description)
+                    _description.key = f"{user.user_id}_{_description.key}"
+                    _description.name = f"{user.username} {_description.name}"
+                    entities.append(
+                        TautulliSessionSensor(
+                            coordinator,
+                            _description,
+                            user.user_id,
```

Comment:
Pass the `user` object instead of the ID, then you do not have to do the filtering in the entity init method

Then you can also store that as `self._user` and use that for the name property and coordinator key

Reply:
I didn't do that before bc I thought it uses more memory in the instance?